src/presets.asm

```diff
@@ -352,7 +344,6 @@ endif
     LDA #$0004 : STA $A7  ; Set optional next interrupt to Main gameplay
 
     JSL $80982A  ; Enable horizontal and vertical timer interrupts
-    JSR $A12B    ; Play 14h frames of music
```

Comment:
OMG we were doing this three times?  Why??  I thought we were supposed to be saving the frames.

I went back in github history, and sure enough back in 2019 we still had this logic where we play 14h frames of music in three places.
https://github.com/tewtal/sm_practice_hack/blob/b26b7aa2901787bd2c28304e07c14ef003321cf7/src/presets.asm

I'm very tempted to put out a 2.3.5 release just for this update.  It would be nice to get some feedback on it to make sure it doesn't break anything.

Reply:
If I load a random preset a couple times very quickly, then go through a door, the screen won't fade in on the other side and it appears to have crashed. I can still open the menu or reload the preset, so it's not really crashed.

Reply:
Interesting.  On the one hand we could really dig into why this happens.  On the other hand we could just prevent users from loading multiple presets within 20 frames of each other, which is sort of the same thing as calling JSR $A12B except less annoying.  If it happens because we are doing too much DMA stuff in the background or something like that, preventing back-to-back preset loads would be a simple fix.

Reply:
I've now managed to crash the audio engine twice by spamming preset loads. We should probably do something to slow this down, but even without a fix, this is still a nice improvement.